modin/backends/base/query_compiler.py

```diff
@@ -573,21 +573,57 @@ def min(self, **kwargs):
         """
         return DataFrameDefault.register(pandas.DataFrame.min)(self, **kwargs)
 
-    def prod(self, **kwargs):
-        """Returns the product of each numerical column or row.
+    def sum(self, squeeze_self, axis, **kwargs):
+        """Returns the sum of each numerical column or row.
```

Comment:
what's the reason for swapping the functions?

Reply:
There is no any reason.

Reply:
in that case pls swap them back to reduce the diff :)

Reply:
swapped